patches/jest-environment-jsdom+29.7.0.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/jest-environment-jsdom/build/index.js b/node_modules/jest-environment-jsdom/build/index.js
+index 2e6c16c..0baf969 100644
+--- a/node_modules/jest-environment-jsdom/build/index.js
++++ b/node_modules/jest-environment-jsdom/build/index.js
+@@ -81,6 +81,10 @@ class JSDOMEnvironment {
+       }
+     );
+     const global = (this.global = this.dom.window);
++    global.fetch = fetch;
++    global.Headers = Headers;
++    global.Request = Request;
++    global.Response = Response;
+     if (global == null) {
+       throw new Error('JSDOM did not return a Window object');
+     }
```

Comment:
Are we not better waiting for official support instead of adding patches ? Maybe I dont understand the concept of patches correctly.

Reply:
Well, usually we do patch packages to modify some behavior.
I have little faith that JSDOM will get fetch support in the near future
https://github.com/jsdom/jsdom/issues/1724
And fetch is a ubiquitous API, it ain't making sense to wait anymore IMO